src/rpdk/core/project.py

```diff
@@ -1323,9 +1327,16 @@ def generate_canary_files(self) -> None:
             not self.file_generation_enabled
             or not Path(self.target_contract_test_folder_path).exists()
         ):
+            LOG.info("Skipping Canary Auto-Generation")
             return
+        LOG.info("Starting Canary Auto-Generation...")
+        if self.file_generation_enabled and self.canary_settings == {}:
+            LOG.warning(
```

Comment:
this log is set at warning level to make the user aware of empty settings resulting in default generation.